blog/2022-07-15-typescript-1.0.0.md

```diff
@@ -0,0 +1,113 @@
+# 1.0.0 release of our TypeScript SDK
+
+After almost 2 years of development, 728 commits, 431 pull requests, and 48 releases we're finally releasing the stable
+version of the Temporal TypeScript SDK.
+
+The SDK was designed with TypeScript-first developer experience in mind, but works equally well with JavaScript.
+
+## What makes this SDK unique?
+
+### Shared Core
+
+This is the first stable SDK built on top of a shared Rust [Core SDK](https://github.com/temporalio/sdk-core/). 
+The development of the Core SDK was started roughly around the time we started to develop the TypeScript SDK and the API
+boundary between the two evolved over time.
+
+By investing in a shared Core, we can reuse lot of the complex concurrency management and state machine logic to build
+new SDKs much faster. For example our - now in Alpha - Python SDK was built in just a few months.
```

Comment:
```suggestion
new SDKs much faster. For example, our Python SDK (currently in Alpha) was built in just a few months.
```
Other possible treatments:
- our now-in-Alpha Python
- our (now in Alpha) Python